src/gui/Preferences.ui

```diff
@@ -2303,6 +2303,20 @@
                  </item>
                 </layout>
                </item>
+               <item>
+                <layout class="QHBoxLayout" name="horizontalLayoutDefaultExportDir">
+                 <item>
+                  <widget class="QLabel" name="labelDefaultExportDir">
+                   <property name="text">
+                    <string>Default Export Directory</string>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QLineEdit" name="lineEditDefaultExportDir"/>
```

Comment:
This UI element also needs to offer a way for users to select a folder using a file dialog. Manually entering a folder name isn't very user friendly